redbeams/src/test/java/com/sequenceiq/redbeams/sync/DBStackJobServiceTest.java

```diff
@@ -0,0 +1,68 @@
+package com.sequenceiq.redbeams.sync;
+
+import com.sequenceiq.redbeams.domain.stack.DBStack;
+import com.sequenceiq.statuschecker.service.JobService;
+import org.junit.Before;
```

Comment:
please use junit from jupiter package

Reply:
This module is using Junit 4

Reply:
I've checked and the jupiter api is added to the module

Reply:
Ahh, I see, this module contains a mixture of Junit 4 and 5 tests.